src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java

```diff
@@ -398,6 +399,13 @@ public String value() {
   private static final String TIMESERIES_TIMEFIELD_AUTO_CONVERSION_DATE_FORMAT_DEFAULT =
       DEFAULT_DATE_TIME_FORMATTER_PATTERN;
 
+  public static final String TIMESERIES_TIMEFIELD_AUTO_CONVERSION_LOCALE_LANGUAGE_TAG_CONFIG =
+      "timeseries.timefield.auto.convert.locale.language.tag";
+  private static final String TIMESERIES_TIMEFIELD_AUTO_CONVERSION_LOCALE_LANGUAGE_TAG_DISPLAY =
+      "The DateTimeFormatter locale language tag too use: Defaults to using the neutral Locale.ROOT.";
+  private static final String TIMESERIES_TIMEFIELD_AUTO_CONVERSION_LOCALE_LANGUAGE_TAG_DOC =
+      "The DateTimeFormatter locale language tag too use with the date pattern: Defaults to Locale.ROOT.";
```

Comment:
[Nit]
too -> to (the same is applicable to line 405 above)

Reply:
Done